Source/UI/PaymentForm/ViewModel/DefaultPaymentViewModel.swift

```diff
@@ -43,6 +43,8 @@ class DefaultPaymentViewModel: PaymentViewModel {
         self.paymentFormStyle = paymentFormStyle
         self.billingFormStyle = billingFormStyle
         self.logger = logger
+        let isCVVOptional = paymentFormStyle?.securityCode == nil
```

Comment:
Does this mean if a user does not enter a CVV then we consider that to be CVV optional by default?

Reply:
This looks at the formStyle, not the user input. The form is provided by the SDK consumer (application), not the user.

What this line of code means is that if the app developer chooses to hide CVV field, our ParsingModel for the inputs (CardCreationModel) will treat its presence as optional. And since there is no UI to enter it, it will work without it.
This allows us if we want to follow Web format of (Show & optional), to furthermore add a property on the actual style that will allow us to show & have optional.